File: GTTS/INL.TrainingService.Models/ISaveTrainingEvent_Result.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
	public interface ISaveTrainingEvent_Result
	{
		Int64 TrainingEventID { get; set; }
		string Name { get; set; }
		string NameInLocalLang { get; set; }
		int TrainingEventTypeID { get; set; }
		string TrainingEventTypeName { get; set; }
		string Justification { get; set; }
		string Objectives { get; set; }
		string ParticipantProfile { get; set; }
		string SpecialRequirements { get; set; }
		int? ProgramID { get; set; }
		int? KeyActivityID { get; set; }
		Int64? TrainingUnitID { get; set; }
		string BusinessUnitAcronym { get; set; }
		string BusinessUnitName { get; set; }
		int? CountryID { get; set; }
		int? PostID { get; set; }
		int? ConsularDistrictID { get; set; }
		int? OrganizerAppUserID { get; set; }
		int? PlannedParticipantCnt { get; set; }
		int? PlannedMissionDirectHireCnt { get; set; }
		int? PlannedNonMissionDirectHireCnt { get; set; }
		int? PlannedMissionOutsourceCnt { get; set; }
		int? PlannedOtherCnt { get; set; }
		decimal? EstimatedBudget { get; set; }
		int? EstimatedStudents { get; set; }
		int? FundingSourceID { get; set; }
		int? AuthorizingLawID { get; set; }
        string Comments { get; set; }

        List<SaveTrainingEventLocation_Item> TrainingEventLocations { get; set; }
		List<SaveTrainingEventUSPartnerAgency_Item> TrainingEventUSPartnerAgencies { get; set; }
		List<SaveTrainingEventProjectCode_Item> TrainingEventProjectCodes { get; set; }
	}
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/isave-training-event_result.ts



import { SaveTrainingEventLocation_Item } from './save-training-event-location_item';
import { SaveTrainingEventUSPartnerAgency_Item } from './save-training-event-uspartner-agency_item';
import { SaveTrainingEventProjectCode_Item } from './save-training-event-project-code_item';

export interface ISaveTrainingEvent_Result {
  
	TrainingEventID: number;
	Name: string;
	NameInLocalLang: string;
	TrainingEventTypeID: number;
	TrainingEventTypeName: string;
	Justification: string;
	Objectives: string;
	ParticipantProfile: string;
	SpecialRequirements: string;
	ProgramID?: number;
	KeyActivityID?: number;
	TrainingUnitID?: number;
	BusinessUnitAcronym: string;
	BusinessUnitName: string;
	CountryID?: number;
	PostID?: number;
	ConsularDistrictID?: number;
	OrganizerAppUserID?: number;
	PlannedParticipantCnt?: number;
	PlannedMissionDirectHireCnt?: number;
	PlannedNonMissionDirectHireCnt?: number;
	PlannedMissionOutsourceCnt?: number;
	PlannedOtherCnt?: number;
	EstimatedBudget?: number;
	EstimatedStudents?: number;
	FundingSourceID?: number;
	AuthorizingLawID?: number;
	Comments: string;
	TrainingEventLocations?: SaveTrainingEventLocation_Item[];
	TrainingEventUSPartnerAgencies?: SaveTrainingEventUSPartnerAgency_Item[];
	TrainingEventProjectCodes?: SaveTrainingEventProjectCode_Item[];
}





